velero-plugins/imagestream/shared.go

```diff
@@ -63,8 +66,11 @@ func getOADPRegistryRoute(uid k8stypes.UID, namespace string, location string, c
 	if oadpRegistryRoute != nil {
 		return *oadpRegistryRoute, nil
 	}
-
-
+	if os.Getenv("OADP_VELERO_POD_REGISTRY_ENABLED") == "true" {
```

Comment:
This can be moved to consuming exported const from OADP-Operator once merged to master.
For now if preferred can  move to const here.